scipy/io/tests/test_mmio.py

```diff
@@ -111,6 +127,137 @@ def test_random_rect_real(self):
         b = mmread(fn)
         assert_array_almost_equal(a,b)
 
+
+class TestMMIOSparseCSR(TestMMIOArray):
+
+    def setUp(self):
+        self.tmpdir = mkdtemp()
+        self.fn = os.path.join(self.tmpdir, 'testfile.mtx')
+
+    def tearDown(self):
+        shutil.rmtree(self.tmpdir)
+    
+    def test_simple(self):
+        a = [[1,2],[3,4]]
```

Comment:
Yes, the test could be simplified. But would it not be better if each test would stay a test of its own. If a test fails, this would simplify finding the error. Nevertheless it would make sense to create a common auxiliary function which would include your described functionality.